boards/catie/zest_core_stm32g474ve/zest_core_stm32g474ve.dts

```diff
@@ -100,8 +100,8 @@ zephyr_udc0: &usb {
 			 <&rcc STM32_SRC_HSE FDCAN_SEL(0)>;
 	pinctrl-0 = <&fdcan3_tx_pb4 &fdcan3_rx_pa8>;
 	pinctrl-names = "default";
-	bus-speed = <125000>;
-	bus-speed-data = <1000000>;
+	bitrate-speed = <125000>;
```

Comment:
I'm not sure, but I read that `bus-speed` is replaced by `bitrate` and not `bitrate-speed`?

Reply:
Resolved in [fae13c1](https://github.com/catie-aq/zephyr_zest-core-stm32g474ve/pull/17/commits/fae13c1fe71a0be6e61d677b567e4e7582099c22).